File: Aula_04/c++/01.cpp
#include <iostream>

class Circulo{
    public:
    double r = 0;

    double calc_area(){
        return 3.14 * (r * r);
    }
};

int main(){
    Circulo c;
    std::cout << "Digite o raio de um circulo: ";
    std::cin >> c.r;
    std::cout << "Area = " << c.calc_area() << std::endl;
    return 0;
}

File: Aula_04/python/01.py
class Circulo:
    def __init__ (self):
        self.r = 0
    def calc_area (self):
        return 3.14 * (self.r * self.r)

c = Circulo()
c.r = float(input("Digite o raio de um circulo: "))
print("Área =", c.calc_area())